packages/admin/cms-admin/src/cronJobs/CronJobs.tsx

```diff
@@ -0,0 +1,113 @@
+import { gql } from "@apollo/client";
+import {
+    LocalErrorScopeApolloContext,
+    MainContent,
+    messages,
+    Stack,
+    Table,
+    TableQuery,
+    Toolbar,
+    ToolbarFillSpace,
+    ToolbarTitleItem,
+    useTableQuery,
+} from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import * as React from "react";
+import { FormattedDate, FormattedMessage, FormattedTime, useIntl } from "react-intl";
+
+import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+import { GQLCronJobsQuery } from "../graphql.generated";
+
+const ScopeIndicatorLabelBold = styled(Typography)`
+    && {
+        font-weight: 400;
+        padding: 0 8px 0 4px;
+        text-transform: uppercase;
+    }
+`;
+
+const ScopeIndicatorContent = styled("div")`
+    display: flex;
+    align-items: center;
+`;
+
+const cronJobsQuery = gql`
+    query CronJobs {
+        cronJobs {
+            id
+            name
+            schedule
+            lastScheduledAt
+        }
+    }
+`;
+
+export function CronJobs(): React.ReactElement {
+    const intl = useIntl();
+
+    const { tableData, api, loading, error } = useTableQuery<GQLCronJobsQuery, undefined>()(cronJobsQuery, {
+        resolveTableData: (data) => ({
+            data: data.cronJobs,
+            totalCount: data.cronJobs.length,
+        }),
+        context: LocalErrorScopeApolloContext,
+    });
+
+    return (
+        <Stack topLevelTitle={intl.formatMessage({ id: "comet.pages.cronJobs", defaultMessage: "Cron Jobs" })}>
+            <ContentScopeIndicator variant="toolbar">
+                <ScopeIndicatorContent>
+                    <Domain fontSize="small" />
+                    <ScopeIndicatorLabelBold variant="body2">
+                        <FormattedMessage {...messages.globalContentScope} />
```

Comment:
Will cron jobs be scoped in the (near) future?

Reply:
No, I don't think so